index.html

```diff
@@ -83,19 +93,36 @@ <h1>My Recent Works</h1>
             src="https://res.cloudinary.com/david79/image/upload/v1674029969/work%20stuff/Img-Placeholder_vhaslg.png"
             alt="Image Placeholder"
           />
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="flex">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-
+          <div class="multi_stories_content">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text ever
+                  since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="flex">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+        <div class="project flex">
+          <div class="project_info flex">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="flex">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
           <button class="btn">See Project</button>
         </div>
         <div class="project flex">
```

Comment:
- [ ] You did a good job creating all the elements required for the desktop version :1st_place_medal: but as a project requirement you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Below I list some issues that I found:

  - The featured project image should take 60% of the page's width and should be aligned to the left, the description should take the remaining 40% and should be aligned to the right :+1: 
  - The featured project should not be overlapping with the regular projects :+1: 

**Your design**

![Screenshot from 2023-01-26 07-53-05](https://user-images.githubusercontent.com/105117832/214844424-5f30d72b-9a0f-4ca6-852f-60e3d1e57a0e.png)

**Figma design**

![Screenshot from 2023-01-26 07-52-45](https://user-images.githubusercontent.com/105117832/214844472-16d30700-9dc4-44b1-99f6-655b44d0ea4d.png)